frontend/tests/badges/BadgesTest.php

```diff
@@ -164,6 +164,22 @@ public function testAllBadges() {
         }
     }
 
+    private static function getBadgesFromArray(array $badgesResults) {
```

Comment:
puedes ponerle tipos a los retornos de las líneas que modificaste?

```suggestion
    private static function getBadgesFromArray(array $badgesResults) -> array {
```

Reply:
Parece que los tipos de retorno se expresan con `:` en vez de `->`.

Reply:
argh, sí. estoy confundiendo la sintaxis de Python y de PHP.